scripts/apps/search/components/preview-subject.tsx

```diff
@@ -24,11 +24,11 @@ export class PreviewSubject extends React.PureComponent<IProps> {
                     /* Adding the Subject heading only for
                     first time so that seems scheme subjects are grouped together.*/
 
-                    subjects.push(<div>
+                    subjects.push(
                         <span
-                            key={subject?.scheme + ':' + subject?.qcode}
+                            key={field.display_name}
```

Comment:
you can probably use `field._id` which should be unique, `display_name` might not be and that would break the UI

Reply:
I also had this thing in my mind but for example, we have one field called `Packages`  if we use `_id ` for that one then on UI we have a label **Service-product**, that's why I have used display_name here so that on UI we have a label **Packages**, is that ok?

Reply:
yep for label it's ok, but this is `key` which is not visible and must be unique